certified-connectors/JG Integrations/apiDefinition.swagger.json

```diff
@@ -0,0 +1,519 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "JG Integrations",
+    "description": "A collection of helper functions to replace missing or paid features in flow.",
+    "version": "1.0",
+    "contact": {
+      "name": "Jack Glasson",
+      "email": "flowconnector@jgsolutions.co.nz",
+      "url": "https://jgsolutions.co.nz"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://jgsolutions.co.nz"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://jgsolutions.co.nz"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files;Collaboration"
+    }
+  ],
+  "host": "integrations.jgsolutions.co.nz",
+  "basePath": "/api/functions",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/crypto/hash_hmac": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "JSON Response",
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "type": "object",
+              "properties": {
+                "hash": {
+                  "type": "string",
+                  "description": "Hashed value encoded using Base64",
+                  "title": "Hashed Value"
+                }
+              }
+            }
+          }
+        },
+        "summary": "HASH HMAC",

```

Comment:
@jglassonnz Can it be in camel case, instead of providing in all the capital letters?